Sources/Send/StateMachine.swift

```diff
@@ -4,13 +4,12 @@
 
 import Foundation
 
-public protocol StateMachine<State, Action, Services> {
```

Comment:
We won't need to define associatedtype in protocol definition within brackets (<>)